backend/capellacollab/alembic/versions/aa747eb041fb_merge_v2_tools_with_v2_project_roles.py

```diff
@@ -0,0 +1,26 @@
+# SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+# SPDX-License-Identifier: Apache-2.0
+
+"""Merge v2/tools with v2/project-roles
+
+Revision ID: aa747eb041fb
+Revises: 3fa75ddfdde8, 6c5d1334d606
+Create Date: 2022-10-17 16:49:03.958375
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "aa747eb041fb"
+down_revision = ("3fa75ddfdde8", "6c5d1334d606")
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    pass
+
+
+def downgrade():
+    pass
```

Comment:
Is it intended that both functions are empty?

Reply:
Yes, it's a alembic merge. It is needed to merge two heads into one, but nothing more :)